lib/tjbot.js

```diff
@@ -199,8 +199,23 @@ TJBot.prototype._setupCamera = function() {
         encoding: 'jpg',
         outputDir: './',
         verticalFlip: this.configuration.see.camera.verticalFlip,
-        horizontalFlip: this.configuration.see.camera.horizontalFlip
+        horizontalFlip: this.configuration.see.camera.horizontalFlip,
+        time: 1
```

Comment:
I can confirm the code path that uses the swizzled method works, it takes like a second and i can see that the `-t 1` option is being included in the call to `raspistill`.  i cannot yet confirm whether the inclusion of `time: 1` in the camera options also causes `-t 1` to be included in the call to `raspistill` (using node-raspistill 0.0.11), due to some trouble i had with npm install including an older version of the library.